### PR TITLE
Create new Rancher service check

### PR DESCRIPTION
**Pull request by kkellerlbl**
Added a new check to see if we're able to create a new service successfully.  This is to try to alert when production narrative stops being able to create new narrative services.

### CODE REVIEW

File: check_rancher_containers.py [hunk 1]
```diff
@@ -133,6 +136,140 @@ def process_section(conf, section):
...
+		newSvcState = session.get(newDummyService['links']['self'], auth=(username,password))
+		dummySvc = newSvcState.json()
+
+		if dummySvc['healthState'] == 'healthy':
+			dummyServiceState = 0
+			dummyServiceStateTxt = 'OK'
+		if dummySvc['healthState'] == 'unhealthy':
+			dummyServiceState = 2
+			dummyServiceStateTxt = 'CRITICAL: created service but service unhealthy'
+		deleteSvcReq = session.delete(newDummyService['links']['self'] , auth=(username,password))
```
Comment (bio-boris) [inline check_rancher_containers.py:267]: unused deleteSvcReq var

File: check_rancher_containers.py [hunk 2]
```diff
@@ -133,6 +136,140 @@ def process_section(conf, section):
 
 #	print (str(memState) + ' ' + envname + '_' + stackname + '_containerMemory - ' + memStateTxt + ' big mem containers: ' + memCommentTxt)
 
+### spin up a dummy new service
+# initially copied from narrative-traefiker
+	container_config = {u'assignServiceIpAddress': False,
```
Comment (bio-boris) [inline check_rancher_containers.py:141]: container_config is snake_case while all other variables are CamelCase

Comment (kkellerlbl) [reply check_rancher_containers.py]: Fixed case (I originally stole that code from narrative-traefiker without changing snake case)

File: check_rancher_containers.py (hunk 1 above)
Comment (kkellerlbl) [reply check_rancher_containers.py:267]: Added check to the delete req (hard to test though)

**Comment by kkellerlbl** [comment]
> This pull request **introduces 1 alert** when merging [e921c8c](https://github.com/kbase/check_mk/commit/e921c8c9a23b2f34607043fe29608e1cfc31c239) into [f9edd44](https://github.com/kbase/check_mk/commit/f9edd44cbb913f900fb84e612937c7273f1d271b) - [view on LGTM.com](https://lgtm.com/projects/g/kbase/check_mk/rev/pr-6b8bfe097f29f81a9143773f4df1ab6ec2a790cc)
> 
> **new alerts:**
> 
> * 1 for Unused import

Ignoring this unused import since pprint does get used during lazy debugging.

**Pull request merged**